.github/workflows/library-release.yml

```
@@ -0,0 +1,77 @@
+name: Build and Release
+
+on:
+  push:
+    tags:
+      - 'v*' # Trigger on version tags (e.g., v1.0.0)
```

I believe this change is required to match the tags that release-plz generates for the core c2pa-rs crate.

```suggestion
      - 'c2pa-v*' # Trigger on version tags (e.g., v1.0.0)
```